packages/parallax/package.json

```diff
@@ -26,11 +26,23 @@
   "maintainers": [
     "Josh Ellis (https://github.com/joshuaellis)"
   ],
+  "scripts": {
+    "dev": "cd ./demo && yarn dev",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "test": "start-server-and-test 'yarn dev' http-get://localhost:3000 'yarn cy:run'"
+  },
   "dependencies": {
     "@react-spring/shared": "~9.2.0",
     "@react-spring/web": "~9.2.0"
   },
   "peerDependencies": {
     "react": "^16.8.0  || ^17.0.0"
+  },
+  "devDependencies": {
+    "@testing-library/cypress": "^7.0.6",
```

Comment:
these dev deps should be in the root of the project, only actual deps are here in a monorepo 👍🏼

Reply:
Got it! Moved them and everything still works.